staging/src/k8s.io/apiserver/pkg/server/statusz/negotiate/negotiation.go

```diff
@@ -25,11 +25,15 @@ import (
 type StatuszEndpointRestrictions struct{}
 
 // AllowsMediaTypeTransform checks if the provided GVK is supported for structured z-page responses.
+// Only JSON and YAML are supported for structured responses.
 func (StatuszEndpointRestrictions) AllowsMediaTypeTransform(mimeType string, mimeSubType string, gvk *schema.GroupVersionKind) bool {
 	if mimeType == "text" && mimeSubType == "plain" {
 		return gvk == nil
 	}
-	return isStructured(gvk)
+	if mimeType == "application" && (mimeSubType == "json" || mimeSubType == "yaml") {
```

Comment:
same comment about keeping the accepted mime-types in the handler switch statement

Reply:
Done